app/data_models/supplementary_data_store.py

```diff
@@ -114,11 +114,31 @@ def get_data(
         For example if you wanted the identifier for the first item in "some_list"
         it would be get_data(identifier="some_list", selectors=["identifier"], list_item_id=list_item_id-1)
         """
-        if self.is_data_repeating(identifier) and not list_item_id:
-            raise InvalidSupplementaryDataSelector(
-                f"Cannot reference items from `{identifier}` outside a repeat"
-            )
+        if not list_item_id and (
```

Comment:
Only minor, but is there any reason why we moved away from the original if statement here which I think is slightly more readable? If we revert the for can be updated to `for _list_item_id in self.list_lookup.get(identifier).values()` as we will know the identifier is in the store

Reply:
Whichever is more readable, I've added back the original one, thanks!